src/common-models/schedule-error-message.model.ts

```diff
@@ -8,10 +8,10 @@ export enum ScheduleErrorLevel {
 }
 export interface ScheduleErrorMessageModel {
   kind: string;
-  title: string;
-  message: string;
+  title?: string;
+  message?: string;
   worker?: string;
   day?: number;
   week?: number;
-  level: ScheduleErrorLevel;
+  level?: ScheduleErrorLevel;
```

Comment:
Ten interfejs dotyczy wszystkich błędów `ScheduleError`: w szczególności może być to `NetworkError`, dla którego trochę bez sensu jest przypisywać te właściwości, bo jest wyświetlany po prostu jako krzyczący trójkąt.